Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Python image to use.
-FROM python:3.9.6-alpine
```

Comment:
Consider using a slimmer base image like `python:3.9.6-slim` to reduce the image size.